.gitlab-ci.yml

```diff
@@ -90,15 +98,52 @@ stable-test:
     - cargo +stable build --verbose --features="${FEATURES}" || exit 1
     - cargo +stable test --workspace --verbose --features="${FEATURES}"
 
-# For now simply collect artifacts
-artifacts:
+build:
   <<:                              *docker-env
   <<:                              *collect-artifacts
-  stage:                           deploy
+  <<:                              *build-refs
+  stage:                           build
   variables:
-    FEATURES:                     "dist-client,dist-server"
+    FEATURES:                     "openssl/vendored,dist-server"
```

Comment:
We probably want to include both client, scheduler and server in the container rather than expecting the user to compile the client himself for a quick test run?

Reply:
Will this be useful for the users, though? The end user will likely want an entrypoint of `sccache`, while the server image should probably use an `sccache-dist` entrypoint (to re-use the image for the scheduler and build server mode).

...speaking of which, I'm always confused by the entrypoint/cmd distinction. @drahnr what would you say will be the most robust/re-usable Dockerfile setup for client/scheduler/server container?

Reply:
I would include all of them, and use a default `CMD` to be `sccache-dist`, so it can be easily overriden. Alt: Create 3 containers, with only different entry points, but all derived from the same base container that includes all tools. No preference here.

Including the client allows people to just do a `{podman,docker} exec -- sccache -s` without having to have any tooling on the host, and the size increase is a couple of MB, so worth for the sake of convenience I'd say.